kafka-ui-api/src/main/java/com/provectus/kafka/ui/emitter/OffsetsInfo.java

```diff
@@ -56,4 +57,8 @@ public boolean assignedPartitionsFullyPolled() {
     return true;
   }
 
+  public Set<TopicPartition> allTargetPartitions() {
+    return Sets.union(nonEmptyPartitions, emptyPartitions);
```

Comment:
wouldn't it better to calculate this in the constructor instead of doing so on every each method call?